frontend/src/v5/ui/routes/dashboard/projects/drawings/uploadDrawingRevisionForm/uploadList/uploadListItem/uploadListItem.component.tsx

```diff
@@ -170,16 +170,27 @@ export const UploadListItem = ({
 				name={fileData.name}
 				size={fileData.size}
 			/>
-			<UploadListItemDestination 
+			<InputController
+				Input={UploadListItemDestination}
 				name={`${revisionPrefix}.drawingName`}
 				key={`${uploadId}.dest`}
+				disabled={disabled}
 				onSelectNewDestination={onClickEdit}
 				index={index}
 				revisionPrefix={revisionPrefix}
-				disabled={disabled}
 				error={!!drawingNameError}
 				helperText={drawingNameError}
 			/>
+			{/* <UploadListItemDestination 
+				name={`${revisionPrefix}.drawingName`}
+				key={`${uploadId}.dest`}
+				onSelectNewDestination={onClickEdit}
+				index={index}
+				revisionPrefix={revisionPrefix}
+				disabled={disabled}
+				error={!!drawingNameError}
+				helperText={drawingNameError}
+			/> */}
 			<InputController
```

Comment:
Upload a revision to a specific drawing and then try to clear the destination. It will crash the app
![image](https://github.com/user-attachments/assets/8fdd211e-29b8-4d6c-b455-0f0dc0e0fa9d)